controllers/gateway/resource.go

```diff
@@ -100,7 +133,7 @@ func (ctx *gatewayContext) makeDeploymentSpec() (*appv1.DeploymentSpec, error) {
 						ImagePullPolicy: corev1.PullAlways,
 						Resources: corev1.ResourceRequirements{
 							Requests: corev1.ResourceList{
-								corev1.ResourceCPU:    apiresource.MustParse("50m"),
+								corev1.ResourceCPU:    apiresource.MustParse("10m"),
```

Comment:
By checking `kubectl top pod`, usually when the gateway POD starts, it only takes `1m`. Lets set `request.cpu=10m` here.